guides/source/active_storage_overview.md

```diff
@@ -113,9 +113,6 @@ To use the test service when testing, you add the following to
 config.active_storage.service = :test
 ```
 
-Continue reading for more information on the built-in service adapters (e.g.
-`Disk` and `S3`) and the configuration they require.
```

Comment:
Not really related but moving this section to the bottom follows the flow better.